_entries/02-07 challenge4.md

```diff
@@ -12,11 +12,20 @@ Ensure the application remains responsive as the number of order submissions inc
 
 #### Run a baseline load test
 
+**Task Hints**
+* A pre-built image on Dockerhub has been created called `bencuk/azch-loadtest`, this uses a tool called 'hey' to inject a large amount of traffic to the capture order API
+* [Azure Container Instances](https://docs.microsoft.com/en-gb/azure/container-instances/) can be used to run this image as a container, e.g using the  `az container create` command.
+* When running as a Container Instance set we don't want it to restart once it has finished, so set `--restart-policy Never` 
+* Provide the endpoint of your capture orders service in `SERVICE_ENDPOINT` environmental variable e.g. `-e SERVICE_ENDPOINT=https://orders.{ingress-ip}.nip.io`
+* You watch the orders come in with the frontend page, and can view the detailed output of the load test with the `az container logs` command
+* Make a note of the results, response times etc
+
 {% collapsible %}
-There is a a container image on Docker Hub ([azch/loadtest](https://hub.docker.com/r/azch/loadtest)) that is preconfigured to run the load test. You may run it in [Azure Container Instances](https://azure.microsoft.com/en-us/services/container-instances/) running the command below
+
+There is a a container image on Docker Hub ([bencuk/azch-loadtest](https://hub.docker.com/r/bencuk/azch-loadtest)) that is preconfigured to run the load test. You may run it in [Azure Container Instances](https://azure.microsoft.com/en-us/services/container-instances/) running the command below
```

Comment:
Can you update to refer to the image at `azch/loadtest`? It is already built with the latest changes.